tests/Doctrine/Tests/ORM/Tools/Pagination/PaginatorTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Tools\Pagination;
+
+use Doctrine\ORM\Query;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+use Doctrine\Tests\Models\Pagination\Company;
+use Doctrine\Tests\Models\Pagination\Department;
+use Doctrine\Tests\Models\Pagination\Logo;
+
+class PaginatorTest extends \Doctrine\Tests\OrmFunctionalTestCase
```

Comment:
What's the reason for creating this file, instead of putting these test cases into PaginationTest?